wire-ios-sync-engine/Source/UserSession/UserSession.swift

```diff
@@ -75,6 +92,10 @@ public protocol UserSession: AnyObject {
     func conversationList() -> ZMConversationList
 
     var ringingCallConversation: ZMConversation? { get }
+
+    var maxAudioLength: TimeInterval { get }
```

Comment:
Is this to limit the length of audio messages? If so, then perhaps we should call this something like `maxAudioMessageLength`?

Reply:
Done!